server/controllers/stripe.js

```diff
@@ -25,25 +30,190 @@ const getRequestPrice = asyncHandler(async (requestId, res) => {
 });
 
 // @route POST /payments/:id/pay
-// @desc Pay pet sitter for a request (after service completed)
+// @desc create payement intent to payout pet sitter for a request (after service completed)
 // @access Private
 exports.createPaymentIntent = asyncHandler(async (req, res, next) => {
   const id = req.params.id;
+  const { customer } = req.body;
   const amount = await getRequestPrice(id, res);
   stripe.paymentIntents.create(
     {
+      customer,
       amount,
       currency: "cad",
       payment_method_types: ["card"],
     },
     (stripeError, stripeResponse) => {
       if (stripeError) {
-        res.status(500).json(stripeError);
+        res.status(500).json({ error: stripeError });
       } else {
         res.status(200).json({
-          success: { clientSecret: stripeResponse.client_secret },
+          clientSecret: stripeResponse.client_secret,
         });
       }
     }
   );
 });
+
+// @route POST /payments/customer
+// @desc create a stripe customer
+// @access Private
+exports.createCustomer = asyncHandler(async (req, res, next) => {
+  const { name, email, phone } = req.body;
```

Comment:
Validate these and return a `400` if any is missing in the request body.

Reply:
> It already has been validated, check the strips.js in the  route folder, validators they were added as middleware 
 ```
 router.route("/customer").post(protect, validateCreateCustomer, createCustomer);